pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/RealtimeTableDataManager.java

```diff
@@ -65,7 +65,7 @@
   private RealtimeSegmentStatsHistory _statsHistory;
   private final Semaphore _segmentBuildSemaphore;
 
-  private static final String STATS_FILE_NAME = "stats.ser";
+  private static final String STATS_FILE_NAME = "statsV2.ser";
```

Comment:
Well, we are really not changing the version of the file (in the sense that the formatting, the fields, etc, are the same), So , I would call it something else altogether. "segment-stats.ser" seems ok.

Also, please add a comment here that says that the old name used to stats.ser and we had to change it when we move the package to be org.apache

Reply:
done